cleanrl/ppo2_continuous_action.py

```diff
@@ -378,7 +439,13 @@ def forward(self, x):
                                 (1.+args.clip_coef) * advantages,
                                 (1.-args.clip_coef) * advantages).to(device)
 
-        policy_loss = - torch.min(ratio * advantages, clip_adv).mean()
+        # Adding entropy bonus: This is computed using the logprobs and the rsampled actions
+        # so the gradients can also be back propagated through it (?)
+        # Does the entropy function of a distribution backpropagate ? Probably not.
```

Comment:
Yes. See 
![image](https://user-images.githubusercontent.com/5555347/76717587-5f094b80-670a-11ea-823b-66c0420a05e7.png)

Reply:
Thanks. I am so lazy, as always hahaha.
Now I know it is possible to easily instanciate a `Variable` to create distributions and such.